### ISSUE TITLE
Pull request 5394 triggered error about systemd in sles11.4 and rhels6.10

### ISSUE THREAD

**Issue by hu-weihua** [xcat2/xcat-core#5433]
In PR #5394, add some configuration for ``systemd``, but it triggered error of ``xcatconfig`` in some lower Linux version, such as sles11.4.and rhels6.10.

The output in sles11.4 are
```
RUN:xcatconfig  -c 2>&1 | tee /tmp/xcatconfig.test [Wed Jul 25 09:29:35 2018]
ElapsedTime:9 sec
RETURN rc = 0
OUTPUT:

Setting up basic certificates.  Respond with a 'y' when prompted.

Existing xCAT certificate authority detected at /etc/xcat/ca, delete? (y/n):# NOTE use "-newkey rsa:2048" if running OpenSSL 0.9.8a or higher
Generating RSA private key, 2048 bit long modulus
...............................+++
..+++
e is 65537 (0x10001)
Using configuration from openssl.cnf
Check that the request matches the signature
Signature ok
Certificate Details:
        Serial Number: 1 (0x1)
        Validity
            Not Before: Jan  1 01:01:01 1970 GMT
            Not After : Jul 25 13:29:36 2038 GMT
        Subject:
            commonName                = xCAT CA
        X509v3 extensions:
            X509v3 Subject Key Identifier:
                36:C0:78:AD:06:0B:56:62:AA:CC:8B:10:4F:C7:D8:99:F9:4F:DD:2D
            X509v3 Authority Key Identifier:
                keyid:36:C0:78:AD:06:0B:56:62:AA:CC:8B:10:4F:C7:D8:99:F9:4F:DD:2D
                DirName:/CN=xCAT CA
                serial:01

            X509v3 Basic Constraints:
                CA:TRUE
            X509v3 Key Usage:
                Certificate Sign, CRL Sign
            Netscape Cert Type:
                SSL CA, S/MIME CA
Certificate is to be certified until Jul 25 13:29:36 2038 GMT (7305 days)
Sign the certificate? [y/n]:

1 out of 1 certificate requests certified, commit? [y/n]Write out database with 1 new entries
Data Base Updated
/root
Created xCAT certificate.
/etc/xcat/cert already exists, delete and start over (y/n)?Generating RSA private key, 2048 bit long modulus
......................................+++
....................................+++
e is 65537 (0x10001)
/root
Using configuration from openssl.cnf
Check that the request matches the signature
Signature ok
Certificate Details:
        Serial Number: 2 (0x2)
        Validity
            Not Before: Jan  1 01:01:01 1960 GMT
            Not After : Jul 20 13:29:37 2038 GMT
        Subject:
            commonName                = c910f02c08p05
        X509v3 extensions:
            X509v3 Subject Alternative Name:
                DNS:c910f02c08p05, DNS:c910f02c08p05
Certificate is to be certified until Jul 20 13:29:37 2038 GMT (7300 days)
Sign the certificate? [y/n]:

1 out of 1 certificate requests certified, commit? [y/n]Write out database with 1 new entries
Data Base Updated
/root
Created xCAT certificate.
/opt/xcat/share/xcat/scripts/setup-dockerhost-cert.sh xcatdockerhost
/etc/xcatdockerca/cert already exists, delete and start over (y/n)?Generating RSA private key, 2048 bit long modulus
..........+++
....................................+++
e is 65537 (0x10001)
/root
Using configuration from openssl.cnf
Check that the request matches the signature
Signature ok
Certificate Details:
        Serial Number: 0 (0x0)
        Validity
            Not Before: Jan  1 01:01:01 1960 GMT
            Not After : Jul 20 13:29:37 2038 GMT
        Subject:
            commonName                = xcatdockerhost
        X509v3 extensions:
            X509v3 Basic Constraints:
                CA:FALSE
            Netscape Cert Type:
                SSL Client, SSL Server, Object Signing
            Netscape Comment:
                OpenSSL Generated Server Certificate
            X509v3 Subject Key Identifier:
                51:25:77:66:CC:B9:E2:7E:AB:83:1B:76:1D:E8:3B:52:E3:3A:1B:70
            X509v3 Authority Key Identifier:
                keyid:01:6A:D3:5D:5D:1E:1F:E8:1E:28:30:F5:CF:E0:88:9A:12:76:9E:6F

            X509v3 Key Usage:
                Digital Signature, Key Encipherment, Key Agreement
            X509v3 Extended Key Usage:
                TLS Web Server Authentication, TLS Web Client Authentication
Certificate is to be certified until Jul 20 13:29:37 2038 GMT (7300 days)

Write out database with 1 new entries
Data Base Updated
/root
/root/.xcat already exists, delete and start over (y/n)?Generating RSA private key, 2048 bit long modulus
..............+++
.................................+++
e is 65537 (0x10001)
Using configuration from openssl.cnf
Check that the request matches the signature
Signature ok
Certificate Details:
        Serial Number: 3 (0x3)
        Validity
            Not Before: Jan  1 01:01:01 1960 GMT
            Not After : Jul 20 13:29:38 2038 GMT
        Subject:
            commonName                = root
        X509v3 extensions:
            X509v3 Basic Constraints:
                CA:FALSE
            X509v3 Key Usage:
                Digital Signature, Key Encipherment, Key Agreement
            X509v3 Extended Key Usage:
                TLS Web Client Authentication
            Netscape Cert Type:
                SSL Client, S/MIME, Object Signing
            Netscape Comment:
                OpenSSL Generated Client Certificate
            X509v3 Subject Key Identifier:
                76:37:C1:B2:B6:99:F6:DE:92:17:E2:39:D1:77:7E:86:AA:01:1A:E7
            X509v3 Authority Key Identifier:
                keyid:36:C0:78:AD:06:0B:56:62:AA:CC:8B:10:4F:C7:D8:99:F9:4F:DD:2D

Certificate is to be certified until Jul 20 13:29:38 2038 GMT (7300 days)
Sign the certificate? [y/n]:

1 out of 1 certificate requests certified, commit? [y/n]Write out database with 1 new entries
Data Base Updated
Created xCAT certificate.
/home/conserver/.xcat already exists, delete and start over (y/n)?Generating RSA private key, 2048 bit long modulus
...............................................................................................................................................................................................................................................................................................................................................................................................+++
............................................................................................+++
e is 65537 (0x10001)
Using configuration from openssl.cnf
Check that the request matches the signature
Signature ok
Certificate Details:
        Serial Number: 4 (0x4)
        Validity
            Not Before: Jan  1 01:01:01 1960 GMT
            Not After : Jul 20 13:29:41 2038 GMT
        Subject:
            commonName                = conserver
        X509v3 extensions:
            X509v3 Basic Constraints:
                CA:FALSE
            X509v3 Key Usage:
                Digital Signature, Key Encipherment, Key Agreement
            X509v3 Extended Key Usage:
                TLS Web Client Authentication
            Netscape Cert Type:
                SSL Client, S/MIME, Object Signing
            Netscape Comment:
                OpenSSL Generated Client Certificate
            X509v3 Subject Key Identifier:
                4D:5E:5C:5C:27:1D:14:9E:75:6C:7C:63:4B:D4:37:0B:BF:CB:FD:26
            X509v3 Authority Key Identifier:
                keyid:36:C0:78:AD:06:0B:56:62:AA:CC:8B:10:4F:C7:D8:99:F9:4F:DD:2D

Certificate is to be certified until Jul 20 13:29:41 2038 GMT (7300 days)
Sign the certificate? [y/n]:

1 out of 1 certificate requests certified, commit? [y/n]Write out database with 1 new entries
Data Base Updated
Created xCAT certificate.
Command failed: systemctl set-environment TERM=vt100 2>&1. Error message: .

Could not set TERM into systemd.
Restarting xcatd ..done
```

**Comment by robin2008** [comment]
In `xcatconfig` codes, it is okay to determine if it is systemd by using the `/usr/lib/systemd/system` directory.  As now xcat ship a unit file by default.

Need to change it to use other method.

**Comment by robin2008** [comment]
To fix this with #5434

**Comment by hu-weihua** [comment]
Have been verified in automation test environment, thanks @robin2008
